features/openhab-addons/src/main/feature/feature.xml

```diff
@@ -270,6 +270,14 @@
     <configfile finalname="${openhab.conf}/services/culintertechno.cfg" override="false">mvn:${project.groupId}/openhab-addons-external/${project.version}/cfg/culintertechno</configfile>
   </feature>
 
+  <feature name="openhab-binding-irtrans" description="IRTrans Binding" version="${project.version}">
+    <feature>openhab-runtime-base</feature>
+    <feature>openhab-runtime-compat1x</feature>
+    <feature>openhab-binding-tcp</feature>
```

Comment:
Yes..... 

Sent from my iPhone

> On 06 Aug 2016, at 14:19, John Cocula <notifications@github.com> wrote:
> 
> In features/openhab-addons/src/main/feature/feature.xml:
> 
> > @@ -270,6 +270,14 @@
> >      <configfile finalname="${openhab.conf}/services/culintertechno.cfg" override="false">mvn:${project.groupId}/openhab-addons-external/${project.version}/cfg/culintertechno</configfile>
> >    </feature>
> >  
> > +  <feature name="openhab-binding-irtrans" description="IRTrans Binding" version="${project.version}">
> > +    <feature>openhab-runtime-base</feature>
> > +    <feature>openhab-runtime-compat1x</feature>
> > +    <feature>openhab-binding-tcp</feature>
> Does the IRTrans binding depend on the TCP binding?
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
>